### ISSUE TITLE
 Unexpected CFLint error during file scan

### ISSUE THREAD

**Issue by bferdinandus** [Pr1st0n/cflint-intellij#32]
Hello I would like to report this error. It happens quire regularly.

This is the log from intellij.
```
Details: java.lang.NullPointerException

java.lang.Throwable: Unexpected CFLint error during file scan
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:179)
	at com.pr1st0n.cflint.CFLintInspection.checkFile(CFLintInspection.kt:52)
	at com.intellij.codeInspection.LocalInspectionTool$1.visitFile(LocalInspectionTool.java:145)
	at com.intellij.extapi.psi.PsiFileBase.accept(PsiFileBase.java:60)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:65)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$7(LocalInspectionsPass.java:346)
	at com.intellij.util.AstLoadingFilter.lambda$toComputable$2(AstLoadingFilter.java:174)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:132)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:121)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:111)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$10(LocalInspectionsPass.java:346)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:136)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1091)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:92)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:104)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:83)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:178)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:83)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool.awaitJoin(ForkJoinPool.java:1708)
	at java.base/java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1039)
	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgress(JobLauncherImpl.java:71)
	at com.intellij.concurrency.JobLauncher.invokeConcurrentlyUnderProgress(JobLauncher.java:49)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.visitRestElementsAndCleanup(LocalInspectionsPass.java:366)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.inspect(LocalInspectionsPass.java:208)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:119)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:56)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:400)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1096)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:393)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:392)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:368)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:178)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:366)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:188)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

This is in my .cflintrc file that I use for this project.
```
{
    "rule" : [ ],
    "excludes" : [
        {"code" : "ARGUMENT_HAS_PREFIX_OR_POSTFIX"},
        {"code" : "ARGUMENT_INVALID_NAME"},
        {"code" : "ARGUMENT_IS_TEMPORARY"},
        {"code" : "ARGUMENT_TOO_LONG"},
        {"code" : "ARGUMENT_TOO_SHORT"},
        {"code" : "ARGUMENT_TOO_WORDY"},
        {"code" : "ARG_DEFAULT_MISSING"},
        {"code" : "ARG_HINT_MISSING"},
        {"code" : "ARG_HINT_MISSING_SCRIPT"},
        {"code" : "ARG_TYPE_ANY"},
        {"code" : "AVOID_USING_ARRAYNEW"},
        {"code" : "AVOID_USING_CREATEOBJECT"},
        {"code" : "COMPLEX_BOOLEAN_CHECK"},
        {"code" : "COMPONENT_HINT_MISSING"},
        {"code" : "COMPONENT_INVALID_NAME"},
        {"code" : "COMPONENT_TOO_LONG"},
        {"code" : "COMPONENT_TOO_WORDY"},
        {"code" : "EXCESSIVE_ARGUMENTS"},
        {"code" : "EXCESSIVE_COMPONENT_LENGTH"},
        {"code" : "EXCESSIVE_FUNCTIONS"},
        {"code" : "EXCESSIVE_FUNCTION_LENGTH"},
        {"code" : "FILE_SHOULD_START_WITH_LOWERCASE"},
        {"code" : "FUNCTION_HINT_MISSING"},
        {"code" : "FUNCTION_TOO_COMPLEX"},
        {"code" : "FUNCTION_TYPE_ANY"},
        {"code" : "FUNCTION_TYPE_MISSING"},
        {"code" : "GLOBAL_VAR"},
        {"code" : "LOCAL_LITERAL_VALUE_USED_TOO_OFTEN"},
        {"code" : "METHOD_INVALID_NAME"},
        {"code" : "METHOD_IS_TEMPORARY"},
        {"code" : "METHOD_TOO_LONG"},
        {"code" : "METHOD_TOO_WORDY"},
        {"code" : "STRUCT_ARRAY_NOTATION"},
        {"code" : "VAR_ALLCAPS_NAME"},
        {"code" : "VAR_HAS_PREFIX_OR_POSTFIX"},
        {"code" : "VAR_INVALID_NAME"},
        {"code" : "VAR_IS_TEMPORARY"},
        {"code" : "VAR_TOO_LONG"},
        {"code" : "VAR_TOO_SHORT"},
        {"code" : "VAR_TOO_WORDY"},
        {"code" : "UNQUOTED_STRUCT_KEY"}
    ],
    "includes" : [

    ],
    "inheritParent" : false
}
```

**Comment by Pr1st0n** [comment]
@bferdinandus thanks for the report!
There is [an open ticket](https://github.com/Pr1st0n/cflint-intellij/issues/30) for this issue. Going to investigate it some when soon.

**Comment by bferdinandus** [comment]
I had seen the other ticket #30 and came to the conclusion that the error I encountered was slightly different.
In ticket #30 the error has to do with the config file `Invalid CFLint Configuration`
In this ticket it is about a error cflint scanning the file `Unexpected CFLint error during file scan`
Mayby it's worh looking into the difference to see if the ticket are really duplicate, or just really similar errors?

**Comment by Pr1st0n** [comment]
I'm pretty sure both issues have similar cause. Need more investigation though

**Comment by Pr1st0n** [comment]
Should be checked separate from https://github.com/Pr1st0n/cflint-intellij/issues/30
